### PR TITLE
Added 3 address lines with add and remove buttons

### PR DESCRIPTION
**Pull request by cyrilbravo**
### Please check if the PR fulfills these requirements:

- [X] After these changes, the app was run and still works as expected
- [X] Tests for these changes were added (if applicable)
- [X] All existing unit tests were run and still pass

### CODE REVIEW

**Comment by cyrilbravo** [comment]
Re-run all the tests and it works now. Can be reviewed now.

File: oop/src/components/AddressInput.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            name="addressLine"
+           class='form-control'
+           :maxlength='maxlength'
+           v-model="localValue" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Input',
```
Comment (harrymaynard-maximus) [inline oop/src/components/AddressInput.vue:14]: Need to change `name` to 'AddressInput'.

File: oop/src/views/MoveInfoPage.vue [hunk 2]
```diff
@@ -241,3 +282,18 @@ export default {
...
+}
+
+#app .remove-button-padding { 
+  padding-left: 20px;
+}
+
+#app .add-remove-button { 
+  min-width: 60px;
+  min-height: 40px;
+}
```
Comment (harrymaynard-maximus) [inline oop/src/views/MoveInfoPage.vue:298]: Replace `<style>` with `<style scoped>` instead. Also can remove `#app` from CSS class declaractions.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

**Comment by cyrilbravo** [comment]
Hi @harrymaynard-maximus , I've added some changes and ready for review.

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good.

**Pull request merged**